### ISSUE TITLE
Gibt es hier noch Support

### ISSUE THREAD

**Issue by bergjet** [Foddy/node-red-contrib-huemagic#393]
Es stehen ja einige Fragen an.

**Comment by uli-rpi** [comment]
Aus meiner Sicht nicht, es gibt seit Monaten keinerlei Antworten von Foody.

**Comment by javos65** [comment]
terrible support, app get worse by the month, turning lights on or off is VERY unreliable, status of lights in NoreRed is mostly incorrect,

**Comment by dprokscha** [comment]
I am using HueMagic since some years and for me it is not unusual that Foddy is not active for some month. Just look into his activity stream... he is alive, but obviously not working on HueMagic. Of course not cool, but don't forget that this project is free and open source. Fork it, fix it or leave it 😉

**Comment by uli-rpi** [comment]
but, a short statemend on his part at the many same error reports would be already helpful.

**Comment by hazymat** [comment]
> but, a short statemend on his part at the many same error reports would be already helpful.

If someone says nothing, they are criticised for not making a short statement. When someone makes a short statement they are criticised for not making a long one. When they make a long statement, they are criticised for not fixing specific problems...

**Comment by Foddy** [comment]
@bergjet ja, den gibt es wieder – auch wenn deine Frage zu dem Zeitpunkt, als du sie gestellt hast, völlig berechtigt war :S

Hier ist lange nichts passiert, das lässt sich nicht schönreden. Inzwischen ist v5.0.0 draußen (auf npm), der komplette Rückstand ist aufgearbeitet und alle offenen Issues sind durchgegangen und beantwortet.

Falls von deinen Fragen von damals noch etwas offen ist, mach gern einen neuen Issue auf – dann bekommst du diesmal auch in vernünftiger Zeit eine Antwort. Danke, dass du überhaupt nachgefragt hast, statt einfach zu gehen.
